docs/reference/commandline/cli.md

```diff
@@ -265,6 +273,15 @@ Following is a sample `config.json` file:
 {% endraw %}
 ```
 
+### Experimental features
+
+To enable [experimental features](https://success.docker.com/article/experimental-features),
+edit the `config.json` file and set `experimental` to `enabled`.
```

Comment:
Should have an example of the syntax, e.g.
```
{ "experimental" : "enabled" }
```

Reply:
Also make sure that we explain that the file must be valid JSON; other options may  already be present in that file, so we could show an example `config.json` with some other ("bogus") options set, e.g.;

```json
{
  "experimental": "enabled",
  "debug": true
}
```